app/build.gradle

```diff
@@ -62,32 +64,32 @@ openApi {
 afterEvaluate {
   // Disable https://github.com/springdoc/springdoc-openapi-gradle-plugin
   // Not particularly useful
-  generateOpenApiDocs.enabled = false
-  forkedSpringBootRun.enabled = false
-
-  tasks.named('forkedSpringBootRun').configure {
-    dependsOn 'bootStartScripts'
-    dependsOn 'checkstyleIntegrationTest'
-    dependsOn 'checkstyleMain'
-    dependsOn 'checkstyleTest'
-    dependsOn 'compileIntegrationTestJava'
-    dependsOn 'dockerfileZip'
-    dependsOn 'integrationTest'
-    dependsOn 'jar'
-    dependsOn 'spotlessJava'
-    dependsOn 'startScripts'
-    dependsOn 'test'
-    // dependsOn ':service-python:pytest' // slows down `:app:build` task
-
-    dependsOn 'jacocoTestReport'
-    dependsOn 'jacocoTestCoverageVerification'
-
-    dependsOn 'dockerPrepare'
-  }
-
-  tasks.named('spectralLint').configure {
-    dependsOn 'forkedSpringBootStop'
-  }
+//  generateOpenApiDocs.enabled = false
+//  forkedSpringBootRun.enabled = false
```

Comment:
according to[ smart people](https://medium.com/codex/clean-code-comments-833e11a706dc), checking in commented code is bad